contracts/Whitelist.sol

```diff
@@ -138,6 +140,15 @@ contract Whitelist is IWhitelist, Ownable, OperatorRole {
    */
   mapping(address => mapping(uint => Lockup)) internal userIdLockups;
 
+ 
+  // error code to be returnd on detectTransferRestriction
+  // returned when from/to jurisdictionId is in halted state
+  uint8 private constant STATUS_ERROR_JURISDICTION_HALT = 4;
```

Comment:
Let's move this up with the other status codes. Constants don't occupy a storage slot so it will not break upgrades.

Reply:
oh, i didn't know that, thanks.